dev/tests/integration/framework/tests/unit/testsuite/Magento/Test/Bootstrap/EnvironmentTest.php

```diff
@@ -9,6 +9,7 @@
  */
 namespace Magento\Test\Bootstrap;
 
+ob_start();
```

Comment:
@joanhe Is there a reason for this? I feel like it definitely does not belong here

Reply:
@bbatsche This is to resolve test failure in php 7.2. ```session_id(): Cannot change session id when headers already sent in ```

Reply:
Maybe try adding this right above the call that's failing, or somewhere in one of the `setUp*()` methods. No code or function calls should be in this file and not inside of the class itself.

Reply:
This is the only place works.